client/db_test.go

```diff
@@ -344,26 +344,27 @@ func TestCommonMethods(t *testing.T) {
 		method string
 	}
 	blacklist := map[key]struct{}{
-		key{batchType, "InternalAddCall"}:    {},
-		key{dbType, "AdminMerge"}:            {},
-		key{dbType, "AdminSplit"}:            {},
-		key{dbType, "NewBatch"}:              {},
-		key{dbType, "Run"}:                   {},
-		key{dbType, "Txn"}:                   {},
-		key{dbType, "GetSender"}:             {},
-		key{txnType, "Commit"}:               {},
-		key{txnType, "CommitInBatch"}:        {},
-		key{txnType, "CommitNoCleanup"}:      {},
-		key{txnType, "Rollback"}:             {},
-		key{txnType, "Cleanup"}:              {},
-		key{txnType, "DebugName"}:            {},
-		key{txnType, "InternalSetPriority"}:  {},
-		key{txnType, "NewBatch"}:             {},
-		key{txnType, "Run"}:                  {},
-		key{txnType, "SetDebugName"}:         {},
-		key{txnType, "SetSnapshotIsolation"}: {},
-		key{txnType, "SetSystemDBTrigger"}:   {},
-		key{txnType, "SystemDBTrigger"}:      {},
+		key{batchType, "InternalAddCall"}:        {},
+		key{dbType, "AdminMerge"}:                {},
+		key{dbType, "AdminSplit"}:                {},
+		key{dbType, "NewBatch"}:                  {},
+		key{dbType, "Run"}:                       {},
+		key{dbType, "Txn"}:                       {},
+		key{dbType, "GetSender"}:                 {},
+		key{txnType, "Commit"}:                   {},
+		key{txnType, "CommitInBatch"}:            {},
+		key{txnType, "CommitNoCleanup"}:          {},
+		key{txnType, "Rollback"}:                 {},
+		key{txnType, "Cleanup"}:                  {},
+		key{txnType, "DebugName"}:                {},
+		key{txnType, "InternalSetPriority"}:      {},
+		key{txnType, "NewBatch"}:                 {},
+		key{txnType, "Run"}:                      {},
+		key{txnType, "SetDebugName"}:             {},
+		key{txnType, "SetSnapshotIsolation"}:     {},
+		key{txnType, "SetSerializableIsolation"}: {},
```

Comment:
I have a slight preference for a single method, which would become a strong preference if we ever support more than two isolation levels.